File: Movies/C++/Park.cpp
#include "Park.hpp"

Park::Park()
{}

Park::~Park()
{}

Park::Park(std::string name, double price, std::string type, std::string time)
{
	this->title = name;
	this->price = price;
	this->time = time;
	this->type = type;
}

std::string Park::getTitle()
{
	return this->title;
}

double Park::getPrice()
{
	return this->price;
}

std::string Park::getTime()
{
	return this->time;
}

std::string Park::getType()
{
	return this->type;
}

File: Movies/Python/Park.py
from Movie import Movie

class Park(Movie):
	def __init__(self, title, price, time, type):
		self.title = title
		self.price = price
		self.time = time
		self.type = type
	
	def getTitle(self):
		return self.title
		
	def getPrice(self):
		return self.price
		
	def getTime(self):
		return self.time
		
	def getType(self):
		return self.type